doc/source/reference/random/extending.rst

```diff
@@ -49,7 +49,7 @@ RNG structure.
     :start-after: example 2
 
 These functions along with a minimal setup file are included in the
-`examples` folder, ``numpy.random.examples``.
+`examples` folder, ``numpy.random._examples``.
```

Comment:
That's still not a directory that the user can access or should know about. All the needed info is already included in the docs, it's the 3 files linked in https://numpy.org/devdocs/reference/random/examples/cython/index.html. So reference that page instead?

Reply:
done